package.json

```diff
@@ -67,7 +67,7 @@
     "through2": "^3.0.1",
     "web-encoding": "^1.1.5",
     "xml": "^1.0.0",
-    "xml2js": "^0.5.0"
+    "xml2js": "^0.4.15"
```

Comment:
because of this edit i have this vulnerability: https://github.com/advisories/GHSA-776f-qx25-q3cc

Reply:
that's a old commit